content/blog/2017/08/2017-08-05-remoting-update.adoc

```diff
@@ -0,0 +1,129 @@
+
+In order to achieve it, I have started the 
+link:https://jenkins.io//projects/remoting/[Remoting sub-project] on Jenkins website. 
+I will also do my best to make regular updates about the Remoting status in blogposts and the link:/mailing-lists[developer mailing list].
+Below you can find updates about incoming changes.
+
+## Remoting Protocols Deprecation
+
+It is almost one year since the release of JNLP4-connect protocol in Remoting 3.0. 
+This protocol has been enabled by default since 2.46.x, and so far it demonstrates good stability being compared to JNLP2 and especially to JNLP3.
+
+At the governance meeting 
+link:http://meetings.jenkins-ci.org/jenkins-meeting/2017/jenkins-meeting.2017-08-02-18.00.html[we decided] to disable old Remoting protocols (JNLP/JNLP2 + CLI1) in new installations by default. 
+There are 3 reasons for it:
+
+* Maintenance of multiple protocols takes much extra efforts.
+The JNL2 NIO engine is complex and barely diagnosable.
+* There are known issues in JNLP2 connection management (see link:/projects/remoting/[the protocol's Errata]). In many cases update to JNLP4 was a resolution
+* JNLP1/JNLP2/CLI1 are unencrypted, and it is not something Jenkins users may expect in 2017
+
+The change will take place in one of the next Weekly releases.
+It is tracked as link:https://issues.jenkins-ci.org/browse/JENKINS-45841[JENKINS-45841] in Jenkins JIRA.
+
+### How?
+
+*  When installation wizard is enabled && it runs in the new installation mode, old protocols will be disabled
+* Jenkins shows an administrative warning when obsolete protocols are enabled
+
+### Compatibility notes
+
+Old instances won't be affected, protocols will be still enabled for them. 
+Newly installed instances launched with the  `-Djenkins.install.runSetupWizard=false` flag (e.g. configuration-as-code) won't be affected as well, 
+the initial configuration is being managed by admins there.
+
+"New" Jenkins instances installed via setup wizard **may be affected** in edge cases. Examples:
+
+* Agents with Remoting older than 3.0 will be unable to connect.
+** **Mitigation:** Before updating make sure Remoting is not bundled custom Docker images, AMIs, etc.
+* Swarm Plugin: old versions of Swarm Client (before 3.3) will be unable to connect to Jenkins, because Remoting 2.x is bundled
+** **Mitigation**: Update Swarm Client
+* **Very** old jenkins-cli.jar without CLI2 support will be unable to connect. 
+** **Mitigation**: Do not use Remoting-based CLI on new instances (see link:/blog/2017/04/11/new-cli/[this blogpost])
+
+## Upgrade to Java 8
+
+Starting from 2.54 Jenkins requires Java 8 to run 
+(link:https://jenkins.io/blog/2017/04/10/jenkins-has-upgraded-to-java-8/[announcement blog post]).
+This version is also required for Jenkins LTS 2.60.1.
+
+Remoting was staying on Java 7 for a while for backporting purposes, 
+but it will be also upgraded to Java 8 in the Remoting 3.11 release.
```

Comment:
it would be good to state when the reader should expect the Remoting 3.11 release